tests/error/comptime_arg_errors/unknown_arg.err

```diff
@@ -0,0 +1,17 @@
+Error: Not known at compile-time (at $FILE:12:12)
+   | 
+10 | def main(b: bool, m: nat) -> None:
+11 |     if b:
+12 |         foo(m)
+   |             ^ Value of this argument must be known at compile-time
+   | 
+10 | def main(b: bool, m: nat) -> None:
+   |                   ------ `m` is a function argument, so its value is not statically
+   |                          known. Consider turning `m` into a comptime argument: `m:
+   |                          @comptime`
+
+Note: We are currently investigating ways to make Guppy's compile-time reasoning
+smarter. Please leave your feedback at
+https://github.com/CQCL/guppylang/discussions/987
```

Comment:
Not sure if this is a good idea?

Reply:
For _this_ example at least, it doesn't seem super relevant